Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
       activemodel (>= 4.2.0)
       kaminari (>= 1.1.0)
       railties (>= 4.2.0)
       sprockets-rails (>= 2.0.0)
-    trestle-active_storage (3.0.0)
+    trestle-active_storage (3.0.1)
       rails (>= 5.2, < 7)
       trestle (~> 0.9.0, >= 0.9.3)
-    trestle-auth (0.4.0)
+    trestle-auth (0.4.1)
       bcrypt (~> 3.1.7)
-      trestle (~> 0.9.0)
+      trestle (~> 0.9.0, >= 0.9.3)
     trestle-tinymce (0.3.0)
       tinymce-rails (~> 5.1)
       trestle (~> 0.9.0)
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.7)
+    tzinfo (1.2.8)
```

Comment:
### [tzinfo](https://tzinfo.github.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.7 → 1.2.8

[_~~change-log~~, [source-code](https://github.com/tzinfo/tzinfo)_]





<details>
<summary>Commits</summary>

A change of **11** commits. See the full changes on [the compare page](https://github.com/tzinfo/tzinfo/compare/v1.2.7...v1.2.8).

These are the first 10 commits:
- (27d96bb) [Update to Ruby 2.4.10.](https://github.com/tzinfo/tzinfo/commit/27d96bb218971a1aafe2d7dd70fad9dfb3f511d9)
- (7299f88) [Stop supporting Rubinius.](https://github.com/tzinfo/tzinfo/commit/7299f88059dbdb5592ea3a27e07f79dee83de9bd)
- (62ae854) [Add Ruby 2.7 on AppVeyor.](https://github.com/tzinfo/tzinfo/commit/62ae85493981fc17e4d497fbd32d8818cd50e908)
- (4edb23c) [Revert "Add Ruby 2.7 on AppVeyor."](https://github.com/tzinfo/tzinfo/commit/4edb23c6ef338446c12c9d4734426c6152fdf460)
- (4b6f072) [Update to Ruby 2.7.2 and JRuby 9.2.13.0.](https://github.com/tzinfo/tzinfo/commit/4b6f0725ecdc1ffc09b33d016e633caedaf63f4e)
- (4433b3e) [Switch to travis-ci.com.](https://github.com/tzinfo/tzinfo/commit/4433b3eca1a40ff047f508268aecdfd590d9b833)
- (5fda4c3) [Remove the deprecated sudo option.](https://github.com/tzinfo/tzinfo/commit/5fda4c3e0f30a5a7405795d327e4373c78b1fd5d)
- (6dfdfc2) [Use up to date Linux distros to test (where possible).](https://github.com/tzinfo/tzinfo/commit/6dfdfc20ebc98125c58442687cfdfa0823312ffd)
- (4b18524) [Support "slim" zoneinfo files produced by default by zic >= …](https://github.com/tzinfo/tzinfo/commit/4b1852416a1f57a2e3766a89835c2d35bc7fde59)
- (196967a) [Update time zone test fixtures based on tzdata 2020d.](https://github.com/tzinfo/tzinfo/commit/196967a9e159930f70563ec824e705da65b9c889)


</details>